File: Bab 8/8.6.2.2 - 2 Character Stair.java
import java.util.Scanner;

public class Main{

	public static void main(String[] args) {
		
		String string = "";
		int baris = 0, kolom = 0, spasi = 0;

		Scanner sc = new Scanner(System.in);

		System.out.println("Masukan angka:\n");

		try{

			string = sc.next();

		}catch(Exception e){

		}

		for (baris = 0; baris < string.length(); baris++)
		{
			for (kolom = 0; kolom < spasi; kolom++)
			{
				System.out.print();
			}

			System.out.println(string[baris]);

			spasi++;
		}
	}
}

File: Bab 8/8.6.2.2 - 2 Character Stair.c
#include <stdio.h>
#include <string.h>

int main()
{
	
	char string[50];
	int baris, kolom, spasi;

	printf("Masukan kata:\n");
	scanf("%s", &string);
	printf("\n");

	spasi = 0;

	for (baris = 0; baris < strlen(string); baris++)
	{
		for (kolom = 0; kolom < spasi; kolom++)
		{
			printf(" ");
		}

		printf("%c\n", string[baris]);

		spasi++;
	}

	return 0;
}